version/version.go

```diff
@@ -4,4 +4,4 @@ const Maj = "0"
 const Min = "10"
 const Fix = "0"
 
-const Version = "0.10.0"
+const Version = Maj + "." + Min + "." + Fix
```

Comment:
Yes let's explicitly write it. Should add this to the coding standards too that repos should have a `version/version.go` structured like this.